lib/phut/setting.rb

```diff
@@ -5,7 +5,7 @@ module Phut
   # Central configuration repository.
   class Setting
     DEFAULTS = {
-      root: File.expand_path(File.join(__dir__, '..', '..')),
+      root: File.expand_path(File.join(File.dirname(__FILE__), '..', '..')),
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.